app/dashboard/models.py

```diff
@@ -2400,6 +2430,136 @@ def is_staff(self):
         """
         return self.user.is_staff if self.user else False
 
+    def calc_activity_level(self):
+        """Determines the activity level of a user
+
+        Returns:
+            str: High, Low, Medium, or New
+
+        """
+        high_threshold = 15
+        med_threshold = 15
+        new_threshold_days = 7
+
+        if self.created_on > (timezone.now() - timezone.timedelta(days=new_threshold_days)):
+            return "New"
+
+        visits = self.actions.filter(action='Visit')
+        visits_last_month = visits.filter(created_on__gt=timezone.now() - timezone.timedelta(days=30)).count()
+
+        if visits_last_month > high_threshold:
+            return "High"
+        if visits_last_month > med_threshold:
+            return "Medium"
+        if visits_last_month > med_threshold:
```

Comment:
should this be `visits_last_month` < `med_threshold`?

Reply:
no; d1574cc644def2763c9e6435530456028a16eb1c